services/src/test/scala/cromwell/services/metadata/impl/WriteMetadataActorSpec.scala

```diff
@@ -33,12 +40,29 @@ class WriteMetadataActorSpec extends ServicesSpec("Metadata") with Eventually {
     }
 
     "Have one event after batch size + 1 is reached" in {
-      1 to 10 foreach { _ => actor ! Action }
+      1 to WriteMetadataActorSpec.BatchRate foreach { _ => actor ! Action }
+      actor.stateName shouldBe WaitingToWrite
```

Comment:
No, this is checking that nothing's changed. If anything we could pause a few seconds then check, I don't like arbitrary long sleeps though. Any ideas?